calico-policies/private-network-isolation/calico-v3/ca-tor/generic-privatehostendpoint.yaml

```diff
@@ -0,0 +1,37 @@
+# **OpenShift 3.11 clusters only**: List all worker node names on your private network.
```

Comment:
Since this region doesn't support Openshift 3.11, I think we can leave this file out